install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -165,6 +165,15 @@ data:
     {{- end }}
     {{- end }}
 
+    # The feature gate
+    {{- if .Values.global.features }}
+    features:
+    {{- range $key, $value := .Values.global.features }}
```

Comment:
We should be adding a proper API for this. This seems no different to what we had before. We were planning on discussing this at the networking wg meeting what the API should be